docker/README.md

```diff
@@ -0,0 +1,25 @@
+# Podman/Docker Compose Examples
+
+You can quickly get started with Polaris by playing with the compose examples provided in this
+directory. The examples are designed to be run with `docker-compose` or `podman-compose`. Each
+example has detailed instructions.
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker/) or [Podman](https://podman.io/docs/installation)
+- [Docker Compose](https://docs.docker.com/compose/install/) 
+  or [Podman Compose](https://docs.podman.io/en/v5.1.1/markdown/podman-compose.1.html)
```

Comment:
Do we need both `docker-compose` and `podman-compose`? I'd prefer to keep `docker-compose` only if there is  no strong reason to introduce `podman`. Any new stack introduced to the projects may add maintenance burden. We should keep it minimum as much as possible.